packages/polaris-viz/src/components/FunnelChartNext/components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,93 @@
+import {Fragment} from 'react';
+import type {Color, DataPoint, LabelFormatter} from '@shopify/polaris-viz-core';
+import {DEFAULT_THEME_NAME} from '@shopify/polaris-viz-core';
+
+import {TOOLTIP_WIDTH} from '../../constants';
+import {FUNNEL_CHART_CONNECTOR_GRADIENT} from '../../../shared/FunnelChartConnector';
+import {FUNNEL_CHART_SEGMENT_FILL} from '../../../shared/FunnelChartSegment';
+import type {FunnelChartNextProps} from '../../FunnelChartNext';
+import {SeriesIcon} from '../../../shared/SeriesIcon';
+import {calculateDropOff} from '../../utilities/calculate-dropoff';
+import {TooltipContentContainer, TooltipTitle} from '../../../TooltipContent';
+
+import styles from './Tooltip.scss';
+
+export interface TooltipContentProps {
+  activeIndex: number;
+  dataSeries: DataPoint[];
+  tooltipLabels: FunnelChartNextProps['tooltipLabels'];
+  labelFormatter: LabelFormatter;
+  percentageFormatter: (value: number) => string;
+}
+
+interface Data {
+  key: string;
+  value: string;
+  color: Color;
+  percent: number;
+}
+
+export function Tooltip({
+  activeIndex,
+  dataSeries,
+  tooltipLabels,
+  labelFormatter,
+  percentageFormatter,
+}: TooltipContentProps) {
+  const point = dataSeries[activeIndex];
+  const previousPoint = dataSeries[activeIndex - 1];
+  const isFirst = activeIndex === 0;
+
+  const dropOffPercentage = Math.abs(
+    calculateDropOff(previousPoint?.value ?? 0, point?.value ?? 0),
+  );
+
+  const data: Data[] = [
+    {
+      key: tooltipLabels.reached,
+      value: labelFormatter(point.value),
+      color: FUNNEL_CHART_SEGMENT_FILL,
+      percent: isFirst ? 100 : 100 - dropOffPercentage,
+    },
+  ];
+
+  if (!isFirst) {
+    data.push({
+      key: tooltipLabels.dropped,
+      value: labelFormatter((previousPoint?.value ?? 0) - (point.value ?? 0)),
+      percent: dropOffPercentage,
+      color: FUNNEL_CHART_CONNECTOR_GRADIENT,
+    });
+  }
+
+  return (
+    <TooltipContentContainer
+      maxWidth={TOOLTIP_WIDTH}
+      theme={DEFAULT_THEME_NAME}
+    >
+      {() => (
+        <Fragment>
+          <TooltipTitle theme={DEFAULT_THEME_NAME}>{point.key}</TooltipTitle>
```

Comment:
Can we add some `aria-labels` in the title and the data row to make it accessible through screen readers? 

e.g. something like this for the data row: ```aria-label={`${key}: ${value}, ${percentageFormatter(percent)}`}```